variables.tf

```diff
@@ -31,11 +27,22 @@ variable "regex_host_allow_pattern_strings" {
 
 variable "ip_rate_limit" {
   description = "The rate limit for IPs matching with a 5 minute window."
-  type        = "string"
+  type        = number
   default     = 2000
 }
 
-variable "ip_set" {
-  description = "ID of IP Set of IP addresses to block."
+variable "ip_sets" {
+  description = "List of sets of IP addresses to block."
+  type        = "list"
```

Comment:
You've got a mix of `"type"` and `type` in there. I'd just pick one and go with it.

Reply:
ok, removed quotes.